.github/workflows/scheduled-tasks.yml

```
@@ -0,0 +1,284 @@
+          # Health check endpoints
+          ENDPOINTS=(
+            "https://api.zippicks.com/health"
+            "https://api.zippicks.com/wp-json/zippicks/v1/health"
+            "https://staging.api.zippicks.com/health"
+          )
+          
+          FAILED=0
+          
+          for endpoint in "${ENDPOINTS[@]}"; do
+            echo "Checking $endpoint..."
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)